src/activities/number_click/Number_click.qml

```diff
@@ -0,0 +1,604 @@
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <https://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.6
+import GCompris 1.0
+
+import "../../core"
+import "number_click.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        source: Activity.url + "background.svg"
+
+        fillMode: Image.PreserveAspectCrop
+        sourceSize.width: Math.max(parent.width, parent.height)
+
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property alias tux: tux
+            property alias numberInLetters: numberInLetters
+            property alias numericNumber: numericNumber
+            property alias errorpanel: errorpanel
+            property alias correctAnswer: correctAnswer
+            property alias wrongAnswer: wrongAnswer
+            property alias firstbar: firstbar
+            property alias secondbar: secondbar
+            property alias thirdbar: thirdbar
+            property alias fourthbar: fourthbar
+            property alias fifthbar: fifthbar
+            property alias sixthbar: sixthbar
+            property alias seventhbar: seventhbar
+            property alias eighthbar: eighthbar
+            property alias ninthbar: ninthbar
+            property alias tenthbar: tenthbar
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        /*Set the main workspace*/
+        Rectangle {
+            id: mainworkspace
+
+            visible : true
+
+            height: parent.height * 0.6
+            width: parent.width * 0.3
+
+            color: "#ABCDEF"
+            border.width: 4
+            border.color: "grey"
+            radius : 30
+
+            anchors.horizontalCenter: parent.horizontalCenter
+            //anchors.horizontalCenterOffset: 100 * parent.width/1080
+            anchors.verticalCenter: parent.verticalCenter
+
+            /*Display the number in letters*/
+            Text {
```

Comment:
Specifically in GCompris, we use GCText class to display text. The main goal is to provide the same font for all the texts.